osu.Game/Screens/Ranking/Expanded/Accuracy/AccuracyCircle.cs

```diff
@@ -263,7 +268,34 @@ protected override void LoadComplete()
 
             using (BeginDelayedSequence(ACCURACY_TRANSFORM_DELAY))
             {
-                double targetAccuracy = score.Rank == ScoreRank.X || score.Rank == ScoreRank.XH ? 1 : Math.Min(1 - virtual_ss_percentage, score.Accuracy);
+                double targetAccuracy = score.Accuracy;
+
+                // Ensure the gauge overshoots or undershoots a bit so it doesn't land in the gaps of the inner graded circle (caused by `RankNotch`es),
+                // to prevent ambiguity on what grade it's pointing at.
+                double[] notchPercentages = { 0.7, 0.8, 0.9, 0.95 };
```

Comment:
The letter grade requirements are hardcoded everywhere in this file so I followed suit. Let me know if I should put these in the same place (in this class) or maybe attempt to centralise this everywhere as a follow-up (not entirely sure where the correct place would be though)

Reply:
Would tend to agree to extracting at least class-local constants for ranks.